libs/src/sdk/api/generated/default/apis/UsersApi.ts

```diff
@@ -570,6 +584,46 @@ export class UsersApi extends runtime.BaseAPI {
         }) as Promise<NonNullable<ConnectedWalletsResponse["data"]>>;
     }
 
+    /**
+     * Gets a user\'s favorite albums
+     */
+    async getFavoriteAlbums(requestParameters: GetFavoriteAlbumsRequest): Promise<NonNullable<FavoritesResponse["data"]>> {
```

Comment:
this seems unrelated? how come we needed this? shouldn't hit have been blocking for a prior commit?

Reply:
yeah I missed a step on my previous change =\

Reply:
didn't realize this was a non-blocking check

Reply:
Yeah I didn't have my full dev setup going and figured either CI or reviewers would tell me if I missed something. The endpoint also worked when I put it up on a node from the PR branch so I thought I was good to go.

Reply:
hmmm yeah we should have failed in CI - looks like we didn't! I'll look into it. Seems like the diffing tool didn't see teh changes

Reply:
Awesome, ty! You probably found it, but https://github.com/AudiusProject/audius-protocol/pull/4397 is the original change. Lmk if you need any more info.

Reply:
yeah it looks like `master` => `main` broke the ci check

Reply:
Ray already fixed - should be good to go moving forward!

Reply:
@raymondjacobson `merge-with-parent` would have saved us here 👀